### PR TITLE
Gabriel/event sign in

### PR DESCRIPTION
**Pull request by gdumouli**
- Signed-up buttons hide/appears if signed-in or not
- Added an indicator on attendee-event-info page to show if a user is signed-in the event

### CODE REVIEW

File: code/noRAM/app/src/main/java/com/example/noram/AttendeeEventInfoActivity.java [hunk 1]
```diff
@@ -37,6 +37,29 @@
  * @author Gabriel
  */
 public class AttendeeEventInfoActivity extends EventInfoActivityTemplate {
+
+    Button signupButton; // button that allows user to sign up for event
```
Comment (ColeDewis) [inline code/noRAM/app/src/main/java/com/example/noram/AttendeeEventInfoActivity.java]: make private

**Review by ColeDewis — APPROVED** [review]
This generally looks fine, but the wording here is a bit unclear to me: is this adding something to display when you **sign up**, or **check in**? It seems like this is for check-in, if i'm reading right, so it might be nice to reword a bit

**Comment by gdumouli** [comment]
> This generally looks fine, but the wording here is a bit unclear to me: is this adding something to display when you **sign up**, or **check in**? It seems like this is for check-in, if i'm reading right, so it might be nice to reword a bit

If a user has signed up (through the "Sign Up" button), I'm hiding the sign-in button and displaying a "Signed-In" indicator on the page. 
I haven't touched checked-in: the event first decide if it display the checked-in or not checked-in page before choosing to show sign-up features or not.
Which wording should I modify to reduce confusion?

**Comment by ColeDewis** [comment]
> > This generally looks fine, but the wording here is a bit unclear to me: is this adding something to display when you **sign up**, or **check in**? It seems like this is for check-in, if i'm reading right, so it might be nice to reword a bit
> 
> If a user has signed up (through the "Sign Up" button), I'm hiding the sign-in button and displaying a "Signed-In" indicator on the page. I haven't touched checked-in: the event first decide if it display the checked-in or not checked-in page before choosing to show sign-up features or not. Which wording should I modify to reduce confusion?

If they've signed up, then let's use the wording as "Signed Up" and not "Signed In" - "Signed In" sounds a lot more like checked in, which isn't what this is doing then.

It also might be nice to be able to un-sign up from the same page if you want to add that - though that could be a separate PR if you want it to

**Comment by gdumouli** [comment]
> > > This generally looks fine, but the wording here is a bit unclear to me: is this adding something to display when you **sign up**, or **check in**? It seems like this is for check-in, if i'm reading right, so it might be nice to reword a bit
> > 
> > 
> > If a user has signed up (through the "Sign Up" button), I'm hiding the sign-in button and displaying a "Signed-In" indicator on the page. I haven't touched checked-in: the event first decide if it display the checked-in or not checked-in page before choosing to show sign-up features or not. Which wording should I modify to reduce confusion?
> 
> If they've signed up, then let's use the wording as "Signed Up" and not "Signed In" - "Signed In" sounds a lot more like checked in, which isn't what this is doing then.
> 
> It also might be nice to be able to un-sign up from the same page if you want to add that - though that could be a separate PR if you want it to

I changed it to signup, I'll do another PR for the "Unsign" button

**Pull request merged**